src/transaction.js

```diff
@@ -772,20 +779,30 @@ Transaction.prototype.runStream = function(query) {
 
 /**
  * Determines whether or not this Transaction should be retried in the event
- * of an ABORTED error.
+ * of a retryable error.
  *
  * @param {error} err - The request error.
  * @return {boolean}
  */
 Transaction.prototype.shouldRetry_ = function(err) {
   return (
-    err.code === ABORTED &&
+    this.isRetryableErrorCode_(err.code) &&
```

Comment:
We need a test in the `shouldRetry_` describe block that checks that `err.code` is passed to `this.isRetryableErrorCode_`.